plugins/ban.py

```diff
@@ -0,0 +1,46 @@
+import requests
+import json
+
+from errbot import BotPlugin, botcmd
+
+
+class Ban(BotPlugin):
+    """
+    Ban from all rooms at once..
+    """
+
+    @botcmd(split_args_with=None)
+    def ban(self, msg, args):
+        """
+        Ban a user from all Gitter rooms at once.
+        corobo ban <@username/username>
+        """
+        user = msg.frm.nick
+        sinner = args[0]
+
+        if sinner.startswith('@'):
+            sinner = sinner[1:]
+
+        joined_rooms = self.bot_config.ROOMS_TO_JOIN
+        headers = {
```

Comment:
Use four level indentation, and the closing brace should be in same column as `h` in `headers`